cmake/CodeFormat.cmake

```diff
@@ -1,8 +1,8 @@
 if(NOT CLANG_FORMAT_EXECUTABLE)
     find_program(CLANG_FORMAT_EXECUTABLE
         NAMES
-        clang-format-4.0
-        clang-format-mp-4.0
+        clang-format-5.0
+        clang-format-5.0
```

Comment:
should this be here twice? is this supposed to be clang-format-mp-5.0?

Reply:
Good catch